orchestration/hca_orchestration/pipelines/cut_snapshot.py

```diff
@@ -77,6 +78,7 @@ def legacy_cut_snapshot_job(hca_env: str, steward: str) -> PipelineDefinition:
             "slack": preconfigure_resource_for_mode(live_slack_client, hca_env),
             "snapshot_config": snapshot_creation_config,
             "dagit_config": preconfigure_resource_for_mode(dagit_config, hca_env),
+            "run_start_time": run_start_time
```

Comment:
Threading in the new run start time resource arg in the legacy cut snapshot job definition.